dbt_project.yml

```diff
@@ -45,30 +45,30 @@ seeds:
   the_tuva_project:
     terminology:
       terminology__calendar:
-        +post-hook: "{{ load_seed('tuva-public-resources/terminology','calendar.csv',compression=true,null_marker=true) }}"
```

Comment:
Is this required for DuckDB to work? We haven't seen issues with keeping the file extension in other adapters.